WebKitBrowser/HtmlApp.config

```diff
@@ -100,12 +100,36 @@ ans(configuration)
 map_append(${configuration} root ${rootobject})
 
 map()
-if(PLUGIN_HTML_APP_MEMORYPRESSURE_WEBPROCESSLIMIT)
-    kv(webprocesslimit ${PLUGIN_HTML_APP_MEMORYPRESSURE_WEBPROCESSLIMIT})
+if(PLUGIN_HTML_APP_MEMORYPRESSURE_WEBPROCESS_SETTINGS_LIMIT OR PLUGIN_HTML_APP_MEMORYPRESSURE_WEBPROCESS_SETTINGS_GPU_LIMIT OR PLUGIN_HTML_APP_MEMORYPRESSURE_WEBPROCESS_SETTINGS_GPU_FILE OR PLUGIN_HTML_APP_MEMORYPRESSURE_WEBPROCESS_SETTINGS_POLLINTERVAL)
+    key(webprocesssettings)
+    map()
+if(PLUGIN_HTML_APP_MEMORYPRESSURE_WEBPROCESS_SETTINGS_LIMIT)
+        kv(limit ${PLUGIN_HTML_APP_MEMORYPRESSURE_WEBPROCESS_SETTINGS_LIMIT})
 endif()
-if(PLUGIN_HTML_APP_MEMORYPRESSURE_NETWORKPROCESSLIMIT)
-    kv(networkprocesslimit ${PLUGIN_HTML_APP_MEMORYPRESSURE_NETWORKPROCESSLIMIT})
+if(PLUGIN_HTML_APP_MEMORYPRESSURE_WEBPROCESS_SETTINGS_GPU_LIMIT)
```

Comment:
I believe you need both to be specified.

Reply:
Sure, gpufile and gpulimit will be set together